bundles/admin/admin-announcements/view/AnnouncementsForm.jsx

```diff
@@ -0,0 +1,163 @@
+              >
+                <Input />
+              </Form.Item>
+              <Form.Item
+                name="content"
+                label={<Message messageKey='newAnnouncement.content' />}
+                rules={[
+                  {
+                    required: true,
+                    message: <Message messageKey='contentError' />,
+                    whitespace: true
+                  }
+                ]}
+              >
+                <TextArea rows={4}/>
+              </Form.Item>
+              <Form.Item
+                name="range_picker"
+                label={<Message messageKey='newAnnouncement.date-range' />}
+                {...rangeConfig}
+              >
+                <DateRange popupStyle={{zIndex: '999999'}} />
+              </Form.Item>
+              <Form.Item name="active" label={<Message messageKey='newAnnouncement.show-popup' />} valuePropName="checked">
+                <Switch onClick={() => controller.toggleActive()}/>
+              </Form.Item>
+              <Row>
+                <Form.Item>
+                  <Button type="primary" htmlType="submit" >
+                    <Message messageKey={'save'}/>
+                  </Button>
+                </Form.Item>
+                <Form.Item>
+                  <Confirm
+                      title={<Message messageKey='messages.deleteAnnouncementConfirm'/>}
+                      onConfirm={() => controller.deleteAnnouncement(index, form.id, form.title)}
+                      okText={<Message messageKey='yes'/>}
+                      cancelText={<Message messageKey='cancel'/>}
+                      placement='top'
+                      popupStyle={{zIndex: '999999'}}
+                  >
+                      <Button
+                    key={key}>
+                          <Message messageKey='delete'/>
+                      </Button>
+                  </Confirm>
+                </Form.Item>
+                <Form.Item>
+                  <Confirm
+                        title={<Message messageKey='messages.cancelAnnouncementConfirm'/>}
+                        onConfirm={() => controller.cancel(index,form.id)}
+                        okText={<Message messageKey='yes'/>}
+                        cancelText={<Message messageKey='cancel'/>}
+                        placement='top'
+                        popupStyle={{zIndex: '999999'}}
+                    >
+                      <Button>
+                        <Message messageKey={'cancel'}/>
+                      </Button>
+                  </Confirm>
+                </Form.Item>
+              </Row>
+            </Form>
+      </div>
+    );
+};
+
+AnnouncementsForm.propTypes = {
+  title: PropTypes.string,
+  key: PropTypes.number,
+  controller: PropTypes.instanceOf(Controller).isRequired
+};
+
+const contextWrap = LocaleConsumer(AnnouncementsForm);
+export { contextWrap as AnnouncementsForm };
```

Comment:
Should work without wrapping to LocaleConsumer as this only uses Message-tag that is a LocaleConsumer itself.
```suggestion
export AnnouncementsForm;
```